modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectActionResourceImpl.java

```diff
@@ -132,6 +144,14 @@ public ObjectAction putObjectAction(
 			Long objectActionId, ObjectAction objectAction)
 		throws Exception {
 
+		if (!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-153714")) &&
```

Comment:
I think we could remove it here because it can be useless since we already blocked it in the POST method. What do you think?